src/integrationTest/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/BlobProcessorTaskTest.java

```diff
@@ -51,17 +51,36 @@
 @SpringBootTest
 public class BlobProcessorTaskTest extends ProcessorTestSuite<BlobProcessorTask> {
 
+    protected static final String VALID_SIGNED_ZIP_FILE_WITH_CASE_NUMBER = "42_24-06-2018-00-00-00.test.zip";
+
     @Before
     public void setUp() throws Exception {
         super.setUp(BlobProcessorTask::new);
     }
 
     @Test
-    public void should_read_from_blob_storage_and_save_metadata_in_database_when_zip_contains_metadata_and_pdfs()
+    public void should_read_blob_and_save_metadata_in_database_when_zip_contains_metadata_and_pdfs()
         throws Exception {
         //Given
         uploadZipToBlobStore(VALID_ZIP_FILE_WITH_CASE_NUMBER); //Zip file with metadata and pdfs
+        testBlobFileProcessed();
```

Comment:
does `VALID_ZIP_FILE_WITH_CASE_NUMBER` contain signature? If not, do we support both signed an unsigned packages?

Reply:
Currently we have to support both (see 
https://tools.hmcts.net/jira/browse/RPE-678).